src/status_im/ui/screens/wallet/components/styles.cljs

```diff
@@ -23,6 +23,13 @@
      :height         52
      :letter-spacing -0.2}))
 
+(def contact-code-text-input
+  {:text-align-vertical :top
```

Comment:
We usually achieve that using parent's container flex properties.
Any pros for this usage?

Reply:
Sorry, I don't get your point. I used this property because multiline text inside of `TextInput` for Android is vertically centered. Are there any alternatives to achieve this?